docs/api/IoT.js-API-File-System.md

```diff
@@ -6,297 +6,588 @@ The following shows fs module APIs available for each platform.
-        - [`fs.rename(oldPath, newPath, callback)`](#fsrenameoldpath-newpath-callback)
-        - [`fs.renameSync(oldPath, newPath)`](#fsrenamesyncoldpath-newpath)
-        - [`fs.stat(path, callback)`](#fsstatpath-callback)
-        - [`fs.statSync(path)`](#fsstatsyncpath)
-        - [`fs.fstat(fd, callback)`](#fsfstatfd-callback)
-        - [`fs.fstatSync(fd)`](#fsfstatsyncfd)
-        - [`fs.write(fd, buffer, offset, length, position, callback)`](#fswritefd-buffer-offset-length-position-callback)
-        - [`fs.writeSync(fd, buffer, offset, length, position)`](#fswritesyncfd-buffer-offset-length-position)
-        - [`fs.writeFile(path, data, [, options], callback)`](#fswritefilepath-data--options-callback)
-        - [`fs.writeFileSync(path, data, [, options])`](#fswritefilesyncpath-data--options)
-        - [`fs.unlink(path, callback)`](#fsunlinkpath-callback)
-        - [`fs.unlinkSync(path)`](#fsunlinksyncpath)
-        - [`fs.close(fd, callback)`](#fsclosefd-callback)
-        - [`fs.closeSync(fd)`](#fsclosesyncfd)
-    - [Class: fs.Stats](#class-fsstats)
-        - [Prototype Functions](#prototype-functions)
-            - [`stats.isDirectory()`](#statsisdirectory)
-            - [`stats.isFile()`](#statsisfile)
+# File System
 
+## Class: fs.Stats
 
-# File System
+fs.Stats class is an object returned from `fs.stat()`,`fs.fstat()` and their synchronous counterparts.
+
+
+
+### `stats.isDirectory()`
+* Returns: {boolean}
+
+Returns true if stated file is a directory.
+
+
+### `stats.isFile()`
+* Returns: {boolean}
+
+Returns true if stated file is a file.
 
+**Example**
+
+```js
+var assert = require('assert');
+var fs = require('fs');
 
-## Module Functions
+fs.stat('test.txt', function(err, stat) {
+  if (err) {
+    throw err;
+  }
+  assert.equal(stat.isFile(), true);
+  assert.equal(stat.isDirectory(), false);
+});
+```
+
+
+## Class methods
+
+### Class method: fs.close(fd, callback)
+* `fd` {integer} File descriptor.
+* `callback` {Function(err: Error | null)} Callback function.
+
+Closes the file of `fd` asynchronously.
+
+**Example**
 
-### `fs.open(path, flags[, mode], callback)`
-* `path <String>` - file path to be opened.
-* `flags <String>` - open flags.
-* `mode <Number>` Default: `0666` - permission mode.
-* `callback <Function(err: Error | null, fd: Number)>`
+```js
+var fs = require('fs');
+
+fs.open('test.txt', 'r', function(err, fd) {
+  if (err) {
+    throw err;
+  }
+  // do something
+  fs.close(fd, function(err) {
+    if (err) {
+      throw err;
+    }
+  });
+});
+```
+
+
+### Class method: fs.closeSync(fd)
+* `fd` {integer} File descriptor.
+
+Closes the file of `fd` synchronously.
+
+**Example**
+
+```js
+var fs = require('fs');
+
+var fd = fs.openSync('test.txt', 'r');
+// do something
+fs.closeSync(fd);
+```
+
+
+### Class method: fs.exists(path, callback)
+* `path` {string} File path to be checked.
+* `callback` {Function(exists: boolean)}
```

Comment:
I think no, we shouldn't. It works like in Node.js, or is there something wrong?